action.yml

```diff
@@ -166,7 +166,7 @@ runs:
         echo "::group::Conda packages building"
         export PYTHON_VERSION="3.${{ inputs.python }}"
         export NUMPY_VERSION=${{ steps.numpy-selection.outputs.numpy_version }}
-        export GIT_DESCRIBE_TAG=$(git describe --tags --abbrev=0)
+        export GIT_DESCRIBE_TAG=""
```

Comment:
Why do we need GIT_DESCRIBE_TAG?

Reply:
To complete: 
- the solution described currently in the guideline does not work: setuptools-scm write the tag in version.py too late during the build : the version used to tag the conda package is the tag n-1.
- the only working alternative is to have double-sourcing by seting manually the version in toml, and use the same for the tag. This may lead to unconsistent version if a tag is set without changing toml or vice-versa